pkg/archive/copy.go

```diff
@@ -67,19 +67,28 @@ func SpecifiesCurrentDir(path string) bool {
 // SplitPathDirEntry splits the given path between its
 // parent directory and its basename in that directory.
 func SplitPathDirEntry(localizedPath string) (dir, base string) {
-	normalizedPath := filepath.ToSlash(localizedPath)
+	var normalizedPath string
+	// On Windows, ToSlash would convert the path to a Linux-style path which
+	// wouldn't work with filepath.* operations. Although it should already
+	// contain \, there is no harm ensuring this by using FromSlash instead.
+	if runtime.GOOS != "windows" {
```

Comment:
It's for the same reason that Linux does ToSlash. FromSlash on Windows will convert forward to back-slashes. Which then means that any filepath.* operations are guaranteed (well, you know what I mean) to work correctly in the correct platform semantics. If I kept it as ToSlash as it is for Linux everything would be using forward slash, and subsequent filepath.* operations would fail on Windows. Does that make sense? (Yes I'm tired too ;)! )